mattermost/mattermost-webapp
# [MM-23716] Re-add create_post write translations

#### Summary
- Looks like a few translations were accidentally removed in this PR https://github.com/mattermost/mattermost-webapp/pull/2517, this PR reverts changes made to `create_post.write` for every language.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-23716

#### Test steps
- Change language to something other than English
- Ensure that the channel text box shows a translated message and not `Write to {{channelName}}`
